scss/_custom.scss

```diff
@@ -107,11 +107,10 @@
 
 .content-container {
   @extend .bg-white;
-  @extend .shadow-sm;
-  @extend .p-4 ;
-  @extend .p-lg-16 ;
+  @extend .py-4 ;
+  @extend .py-lg-16;
```

Comment:
Presuming that there does not need to be further adjustments to previous displays that had the x-axis padding, then all of this should suffice:
![image](https://user-images.githubusercontent.com/6549759/156433264-8c5e299f-1895-4203-8c74-e3ec90e50adc.png)